File: output/ChatGPT/codenet/C++/Java/s778828888.java
import java.util.Scanner;

public class s778828888 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n, m, x;
        int[] a = new int[105];

        n = scanner.nextInt();
        m = scanner.nextInt();
        x = scanner.nextInt();

        for (int i = 0; i < m; ++i) {
            int temp = scanner.nextInt();
            a[temp] = 1;
        }

        int ansl = 0, ansr = 0, i;
        for (i = x; i <= n; ++i) {
            ansr += a[i];
        }
        for (i = x; i >= 0; --i) {
            ansl += a[i];
        }

        System.out.println(Math.min(ansl, ansr));
    }
}

File: output/ChatGPT/codenet/C++/Python/s778828888.py
n, m, x = map(int, input().split())
a = [0] * 105

for i in range(m):
    temp = int(input())
    a[temp] = 1

ansl = 0
ansr = 0
i = x

while i <= n:
    ansr += a[i]
    i += 1

i = x

while i >= 0:
    ansl += a[i]
    i -= 1

print(min(ansl, ansr))